js_modules/dagit/packages/core/src/instance/RepoFilterButton.tsx

```diff
@@ -16,7 +16,7 @@ export const RepoFilterButton: React.FC = () => {
         style={{width: 'auto'}}
         onClose={() => setOpen(false)}
       >
-        <DialogHeader icon="repo" label="Repositories" />
+        <DialogHeader icon="repo" label="Filter code locations" />
```

Comment:
When filtering repositories for the left nav (and other filtered locations, e.g. Overview tabs), use "code locations" as the noun instead. This isn't entirely accurate right now, since users may still have locations with N repositories and we're showing a flat list of repositories here, but we kind of need some noun here, and this seems directionally correct.